src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/CcdCaseRetriever.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+@Service
+public class CcdCaseRetriever {
+    public static final String CASE_TYPE_ID = "Bulk_Scanned";
+    private final CoreCaseDataApi coreCaseDataApi;
+
+    CcdCaseRetriever(CoreCaseDataApi coreCaseDataApi) {
+        this.coreCaseDataApi = coreCaseDataApi;
+    }
+
+    CaseDetails retrieve(CcdAuthInfo authInfo, String caseRef) {
```

Comment:
retrieve --> retrieveCCDCase or  retrieveCaseData?

Reply:
nah  I think that is overboard .. its retrieve on the CcdCaseRetriever so the context explains the naming.